packages/polaris-viz/src/components/LineChart/Chart.tsx

```diff
@@ -126,14 +125,19 @@ export function Chart({
 
   const [annotationsHeight, setAnnotationsHeight] = useState(0);
 
+  const containerDimensions = {
+    height: containerBounds?.height ?? 0,
+    width: containerBounds?.width ?? 0,
+  };
+
   const {legend, setLegendDimensions, height, width} = useLegend({
     data: [
       {
         shape: 'Line',
         series: data,
       },
     ],
-    dimensions,
+    dimensions: containerDimensions,
```

Comment:
This line used to be dimensions and so did line 261, so I am a bit confused why they are replaced with different variables, not the same one. Is this intentional? Were we using the wrong variable before?

Reply:
This is intentional. This was using the original `dimensions` prop that was actually bounds.

Now `containerDimensions` only passed `height` and `width`.